openkfw/TruBudget
# api, frontend: added workflow mode option

### Checklist




### Description


### How to test

1. Login as mstein
2. Create a new subproject and from dropdown Workflow Mode - Unordered
3. Create a couple of wf items
4. You should be able to close them in any order
5. Ordered WF mode should work the same as before - closing possible only in order

#### Excel-export test

1. Export data
2. Subproject sheet should have wf mode (localized)




Closes #1829
